syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.messages";
option java_outer_classname = "Messages";
option objc_class_prefix = "MSG";

package messages;

service RuntimeStats {
    rpc GetStats(StatsRequest) returns (StatsResponse) {}
}

message StatsRequest {
    int64 timestamp = 1; 
}

message StatsResponse {
    int64 timestamp = 1;
    int32 mem_usage = 2;
    int32 time_online = 3;
}